### DISCUSSION TITLE
v0.30.5

### DISCUSSION THREAD

**Discussion by atc0005** [atc0005/check-vmware#723] [Stable Release]
## Overview

- Bug fixes
- Dependency updates
- GitHub Actions Workflows updates
- built using Go 1.19.1
  - Statically linked
  - Linux x64

## Added

- (GH-720) Add Vulnerability Analysis GitHub Actions Workflow

## Changed

- Dependencies
  - `Go`
    - `1.17.13` to `1.19.1`
  - `rs/zerolog`
    - `v1.27.0` to `v1.28.0`
  - `google/go-cmp`
    - `v0.5.8` to `v0.5.9`
  - `github/codeql-action`
    - `v2.1.18` to `v2.1.22`
- (GH-710) Update project to Go 1.19
- (GH-712) Update Makefile and GitHub Actions Workflows
- (GH-719) Update check_vmware_hs2ds2vms plugin to list all hosts and
  datastores missing Custom Attributes

## Fixed

- (GH-711) Adjust doc comments formatting for doc.go
- (GH-714) vsphere.GetObjectCustomAttribute func (incorrectly) uses fixed
  object type in log/error messages

## References

- See the [`v0.30.5` milestone](https://github.com/atc0005/check-vmware/milestone/60?closed=1) for the issues associated with this release.
- See our [CHANGELOG](https://github.com/atc0005/check-vmware/blob/master/CHANGELOG.md) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/atc0005/check-vmware/releases/tag/v0.30.5'>v0.30.5</a>.</em>

**Comment by aucompbiker** [comment]
Adam,

Not sure if you're aware but the individual asset files were not generated for this release.

**Reply by atc0005** [reply]
@aucompbiker Thanks for dropping a note. In this case I'm aware (ran out of time this AM).

Should be uploaded soon.

**Reply by atc0005** [reply]
@aucompbiker assets are available for download now.
